spec.md

```diff
@@ -1349,15 +1432,22 @@ can be mounted on all app instances simultaneously.
 
 If `"bindings_retrievable" :true` is declared for a service in the
 [Catalog](#catalog-management) endpoint, Service Brokers MUST support this
-endpoint for all plans of the service.
+endpoint for all services and plans that support bindings (`"bindable": true`)
+and this endpoint MUST be available immediately after the
+[Polling Last Operation for Service Bindings](#polling-last-operation-for-service-bindings)
+endpoint returns `"state": "succeeded"` for a [Binding](#binding) operation.
+Otherwise, Platforms SHOULD NOT attempt to call this endpoint under any
```

Comment:
why not?  Wouldn't they just get a 404 if its not ready/there?

Reply:
removed